workaround for $MPI_CXX_LIBRARIES being empty and CMake tripping over that with:
CMake Error at CMakeLists.txt:252 (list):
  list GET given empty list
author: Kenneth Hoste (HPC-UGent)
--- sissopp/CMakeLists.txt.orig	2022-06-23 17:17:12.083398990 +0200
+++ sissopp/CMakeLists.txt	2022-06-23 17:17:26.326311818 +0200
@@ -248,9 +248,9 @@
 # Check MPI
 find_package(MPI REQUIRED)
 include_directories(${MPI_CXX_INCLUDE_DIRS})
-set(MPI_LIBRARIES, ${MPI_CXX_LIBRARIES})
-list(GET MPI_CXX_LIBRARIES 0 MPI_LIBRARY)
-get_filename_component(MPI_DIR ${MPI_LIBRARY} DIRECTORY)
+#set(MPI_LIBRARIES, ${MPI_CXX_LIBRARIES})
+#list(GET MPI_CXX_LIBRARIES 0 MPI_LIBRARY)
+#get_filename_component(MPI_DIR ${MPI_LIBRARY} DIRECTORY)
 
 # Build libnlopt
 if(BUILD_PARAMS)
